recipes/cfitsio/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set name = "cfitsio" %}
+{% set version = "3.410" %}
+{% set nodotversion = "3410" %}
+{% set sha256 = "a556ac7ea1965545dcb4d41cfef8e4915eeb8c0faa1b52f7ff70870f8bb5734c" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}{{ nodotversion }}.tar.gz
+  url: ftp://heasarc.gsfc.nasa.gov/software/fitsio/c/{{ name }}{{ nodotversion }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  detect_binary_files_with_prefix: true
+  skip: true  # [win]
+
+requirements:
+  build:
+    - toolchain
+
+test:
+  commands:
+    - test -f $PREFIX/lib/libcfitsio.a
+
+about:
+  home: http://heasarc.gsfc.nasa.gov/fitsio/fitsio.html
+  license: Unspecified
```

Comment:
In theory, because this is a government sponsored software, it should be public domain (even when they add another open source license). However, it would be nice to get in touch with them and ask them to specify something here.